docs/tutorials/network-and-buffering-config.md

```diff
@@ -31,7 +31,7 @@ So, for example, if the base delay is 1s, and the backoff factor is 2:
 4. delay of 4, retry at t = (3 + 4) = 7
 5. delay of 8, retry at t = (7 + 8) = 15
 
-and so on.  To avoid many clients hammering a server at the same exact time,
+and so on.  To avoid many clients hammering a server at the same time,
```

Comment:
I'd keep the "exact". It makes it clearer what the problem is.

Reply:
Isn't "exact same" bringing redundancy? 

For example - 

- *squash the commits together*
- __*squash the commits*__